packages/core/src/App/Containers/Layout/header.jsx

```diff
@@ -198,7 +198,7 @@ export default connect(({ client, common, ui, modules }) => ({
     enableApp: ui.enableApp,
     header_extension: ui.header_extension,
     is_acc_switcher_disabled: ui.is_account_switcher_disabled,
-    is_acc_switcher_on: ui.is_accounts_switcher_on,
+    is_acc_switcher_on: !!ui.is_accounts_switcher_on,
```

Comment:
I think this change is not necessary because the `is_account_switcher_on` has to be a boolean always. if there is a case that this value is not a boolean, I think that's a bug we have to probe.

Reply:
Yes, I'm not sure why it was changing from `bool` to something else, but that's what the warning message was saying, so I just enforced it as a bool here since we are using it only as a `boolean` in Header -> AccountsInfo -> AccountsSwitcher. 
I didn't have time to debug this completely.